### PR TITLE
partial def for leaflet module

### PR DESCRIPTION
**Pull request by turadg**
Partial definition for http://leafletjs.com/reference-1.2.0.html

### CODE REVIEW

File: definitions/npm/leaflet_v1.2.0/flow_all/leaflet_v1.2.0.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+// flow-typed signature: 9eac0d0d9e2129cc3d92d8f4f8c072db
+// flow-typed version: <<STUB>>/leaflet_v1.2.0/flow_v0.69.0
```
Comment (GAntoine) [inline definitions/npm/leaflet_v1.2.0/flow_all/leaflet_v1.2.0.js]: Remove these 2 lines.

**Comment by turadg** [comment]
Looking to merge as partial. I covered what we needed and want to share that for people to build on.

Anything you'd like to see in the code as caveat? (btw, feel free to edit in place.)

**Comment by TrySound** [comment]
@turadg Did you see this one?
https://github.com/flowtype/flow-typed/pull/1074

**Comment by turadg** [comment]
Shoot, nope! I searched Google and for files in master.

Why not merge that one? Seems like it would be valuable even as is. At the least to unify efforts to improve leaflet typing. :)

Incidentally, it looks like https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/leaflet/index.d.ts is very complete for v1.2. I tried https://github.com/joarwilk/flowgen but it has some bugs. The output can be manually fixed, but then there's the dependency on `geojson` which also is in DefinitelyTyped but not flow-typed.

**Comment by TrySound** [comment]
If you'd like to help with that PR, ask Michael. I'm sure he will agree to give an access to his fork.

**Comment by RumyantsevMichael** [comment]
Yes, I will

**Pull request closed without merging**